server/handlers/component_handler.go

```diff
@@ -1554,6 +1640,11 @@ func RegisterEntity(content []byte, entityType entity.EntityType, h *Handler) er
 		helpers.HandleError(v1beta1.Host{
 			Hostname: r.Model.Registrant.Hostname,
 		}, &r, err, isModelError, isRegistrantError)
+		return err
+	case entity.Model:
+		//handler oci return here
+		return nil
+	case entity.PolicyDefinition:
```

Comment:
Is this a placeholder for future support? Add a comment indicating this, if so.

If not, it's out of place and not something that we currently have and should be removed or replaced with Relationship, if that is what is meant.